vgm.sh

```diff
@@ -17,6 +17,12 @@ cmake "$SOURCEDIR" \
 
 make ${JOBS+-j $JOBS} install
 
+# Relocation of .cmake files
+for CMAKE in $(find "$INSTALLROOT/lib" -name '*.cmake'); do
+  sed -ideleteme -e "s!$ROOTSYS!\$ENV{ROOTSYS}!g; s!$G4INSTALL!\$ENV{G4INSTALL}!g" "$CMAKE"
+done
+find "$INSTALLROOT/lib" -name '*deleteme' -delete || true
+
```

Comment:
`find "$INSTALLROOT/lib" -name '*.cmake' -exec sed -e "s!$ROOTSYS!\$ENV{ROOTSYS}!g; s!$G4INSTALL!\$ENV{G4INSTALL}!g" {} \;`, no?

Reply:
I am not author of the above; yours looks simpler. But again, the change should go into master, first.